drf_react_template/schema_form_encoder.py

```diff
@@ -413,7 +413,11 @@ def _set_validation_properties(
         ]
         if custom_validators:
             result['ui:custom-validators'] = [
-                {'code': v.code, 'message': v.message} for v in custom_validators
+                {
+                    'code': getattr(v, 'code'),
```

Comment:
We don't want to set a default of none. Please set to validation class name as per previous PR comment. Thanks.

Reply:
I've updated It. I had disabled it while doing some tests. My bad